docs/pages/components/side-navigation.md

```diff
@@ -140,10 +143,10 @@ On expand, the `is-expanded` class should be propagated also to `__content` elem
             </li>            
             <li class="fd-nested-list__item">
                 <div class="fd-nested-list__content has-child" tabindex="0">
-                    <a class="fd-nested-list__link" href="#" tabindex="-1">
+                    <a class="fd-nested-list__link" role="button" tabindex="-1">
```

Comment:
*could* this be a button?

i could have sworn that IE handles `<a>` without `href` values badly

Reply:
I figured out a better setup for this to use an actual button.